ext/zip/config.m4

```diff
@@ -38,6 +38,8 @@ if test "$PHP_ZIP" != "no"; then
 
   AC_DEFINE(HAVE_ZIP,1,[ ])
 
+  PHP_EVAL_INCLINE($LIBZIP_CFLAGS)
+
```

Comment:
This approach is correct. But if you're going to add this, please place it two lines after `PKG_CHECK_MODULES`, e.g.:

`PKG_CHECK_MODULES`
`(blank line)`
`PHP_EVAL_INCLINE($LIBZIP_CFLAGS)`